app/src/main/java/com/android/streetworkapp/ui/tutorial/TutorialEvent.kt

```diff
@@ -0,0 +1,213 @@
+  val pagerState = rememberPagerState(pageCount = { 5 })
+
+  // Tutorial text for each page
+  val tutoText1 = buildAnnotatedString {
+    withStyle(style = SpanStyle(color = Color.Red)) { append("Click") }
+    append(" on your favorite park")
+  }
+
+  val tutoText2 = buildAnnotatedString {
+    withStyle(style = SpanStyle(color = Color.Green)) { append("Look") }
+    append(" for others events in the park\nOr ")
+    withStyle(style = SpanStyle(color = Color.Red)) { append("start") }
+    append(" one yourself")
+  }
+
+  val tutoText3 = buildAnnotatedString {
+    withStyle(style = SpanStyle(color = Color.Green)) { append("Join") }
+    append(" an already ongoing event")
+  }
+
+  val tutoText4 = buildAnnotatedString {
+    append("Or ")
+    withStyle(style = SpanStyle(color = Color.Red)) { append("create") }
+    append(" your own event")
+  }
+
+  Box(modifier = Modifier.fillMaxSize().testTag("tutoScreenBoxContainer")) {
+    Column(
+        modifier =
+            Modifier.fillMaxWidth().testTag("tutoScreenColumnContainer").padding(top = 40.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally) {
+          Box(
+              modifier = Modifier.fillMaxSize(0.9f).testTag("tutoPageContainer"),
+          ) {
+            HorizontalPager(state = pagerState) { page ->
+              // Display the corresponding page
+              when (page) {
+                0 -> Page1()
+                1 ->
+                    PageTuto(
+                        R.drawable.tuto_map, tutoText1, "tutoColumn1", "tutoImage1", "tutoText1")
+                2 ->
+                    PageTuto(
+                        R.drawable.tuto_park, tutoText2, "tutoColumn2", "tutoImage2", "tutoText2")
+                3 ->
+                    PageTuto(
+                        R.drawable.tuto_join, tutoText3, "tutoColumn3", "tutoImage3", "tutoText3")
+                4 -> {
+                  Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.BottomEnd) {
+                    PageTuto(
+                        R.drawable.tuto_event, tutoText4, "tutoColumn4", "tutoImage4", "tutoText4")
+                    PageEndButton(navigationActions)
+                  }
+                }
+              }
+            }
+          }
+
+          // Dots indicator
+          Row(
+              horizontalArrangement = Arrangement.Center,
+              verticalAlignment = Alignment.CenterVertically,
+              modifier = Modifier.padding(16.dp).testTag("tutoDot")) {
+                repeat(5) { index ->
+                  val isSelected = pagerState.currentPage == index
+                  Box(
+                      modifier =
+                          Modifier.size(if (isSelected) 12.dp else 12.dp)
+                              .padding(4.dp)
+                              .clip(CircleShape)
+                              .background(if (isSelected) Color.Blue else Color.Gray))
```

Comment:
Improvement: Use the color defined in the `ColorPalette` in the `Color.kt` file in this case I think you should use `INTERACTION_COLOR_DARK` and `BORDER_COLOR`